auto_round/autoround.py

```diff
@@ -449,6 +457,14 @@ def calib(self, nsamples, bs):
                 for key in data.keys():
                     data_new[key] = data[key].to(self.model.device)
                 input_ids = data_new["input_ids"]
+            elif isinstance(data, tuple) or isinstance(data, list):
+                if self.multimodal:
+                    data_new = {"input_ids": data[0].to(self.model.device), \
```

Comment:
better not add key 'multimodal',  only support dict if there are multiples inputs?